packages/wonder-blocks-toolbar/src/components/toolbar.js

```diff
@@ -104,19 +105,22 @@ export default class Toolbar extends React.Component<Props> {
                     size === "small" && sharedStyles.small,
                 ]}
             >
-                <View style={sharedStyles.column}>
-                    <View style={sharedStyles.leftColumn}>{leftContent}</View>
+                <View
+                    style={[
+                        sharedStyles.column,
+                        sharedStyles.leftColumn,
+                        title ? sharedStyles.withTitle : null,
+                    ]}
```

Comment:
**explanation:** Decided to refactor this part to remove one extra `div` in the toolbar (did the same thing with `rightContent`)